examples/storage-examples/ceph-examples/mysql_ceph_plugin/README.md

```diff
@@ -0,0 +1,236 @@
+Example 2: Using the Ceph-RBD Plugin
+====================================
+
+This example uses the ceph/rbd plugin directly in the pod spec. Make sure that any mysql pods created in other examples have been deleted before continuing with this example:
+
+```
+#on the OSE-master:
+$ oc get pods
+
+#if you see a mysql pod above:
+$ oc delete pod <pod-name>
+```
+
+### Environment:
+If the steps to install the environment, ceph, ose, and mysql have not already been completed, then follow the instuctions linked-to directly below:
+
+The enviromnent used for all of the examples is described [here](../ENV.md). It is assumed that ceph is already up and running, either on bare metal, in a VM, or containerized.
+
+### Setting up Openshift for Ceph:
+The steps needed to setup a simple OSE cluster that supports ceph are described [here](../OSE.md).
+
+### Setting up MySQL:
+Follow the instructions [here](../MYSQL.md) to initialize and validate containerized mysql.
+
+### Defining the Pod Spec File:
+This example defines a simple [pod spec](mysql-ceph-pod.yaml) using the same mysql image, a container volume mount which references the ceph-rbd persistent storage plugin, and the ceph monitor's ip address.
+
+The pod created by this spec needs to be **privileged**, as seen in the container's "securityContext" section. This is necessary for the rbd mount to succeed. If the container is not privileged and if the OSE-node has selinux set to *enforcing* then this error will be seen:
+
+```
+#on the target OSE-node:
+$ docker logs 45f023ef10ce  #mysql container-ID
+chown: cannot read directory `/var/lib/mysql/': Permission denied
+```
+
+By setting selinux to permissive on *each* OSE-node the above error is also aboided. However, running the mysql container privileged is likely preferred.
+
+On the OSE-master, create the mysql pod:
+
+```
+#on the OSE-master:
+$ oc create -f mysql-ceph-pod.yaml 
```

Comment:
This applies also to ceph-examples/mysql_ceph_pvc and ceph-examples/mysql_ceph_template